packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -399,19 +399,41 @@ const defaultConfigs = {
     expression: "10 * DEFI_PULSE_SUSHI_TVL / DEFI_PULSE_UNISWAP_TVL"
   },
   CNYUSD: {
-    type: "tradermade",
-    pair: "CNYUSD",
-    minTimeBetweenUpdates: 600,
-    minuteLookback: 7200,
-    hourlyLookback: 604800,
-    ohlcPeriod: 10 // CNYUSD only available at 10 minute granularity
+    type: "fallback",
+    orderedFeeds: [
+      {
+        type: "tradermade",
+        pair: "CNYUSD",
+        minTimeBetweenUpdates: 600,
+        minuteLookback: 7200,
+        hourlyLookback: 604800,
+        ohlcPeriod: 10 // CNYUSD only available at 10 minute granularity
+      },
+      {
+        type: "forexdaily",
+        base: "CNY",
+        symbol: "USD",
+        lookback: 604800
```

Comment:
Similar question as above -- could make sense to shorten this to 1 or 2 days (if the feed is daily). We basically want to ensure we _at least_ cover the standard 2 hour period, but any more than that seems excessive. But maybe I'm missing something?

Reply:
That's reasonable, I'm going to set to 3 days for the reason mentioned before